content/faq/how-to-publish.md

```diff
@@ -0,0 +1,39 @@
+## How to publish content on LBRY
+
+Advanced users: All of the steps below are for the LBRY app.  For additional details of how to conduct any of these actions through the cli please see https://lbryio.github.io/lbry/cli/
+
+### How to create a channel
+1. Download the LBRY app from lbry.io/get. If you already have the application and it prompts you to update, run the update to be on the most recent version.
+2. Load the application and  click the `Publish` button.
+3. In the `Channel Name` section click the dropdown and select `New Channel` and  declare the name you would like for your channel.  For more details on different channel types, see https://lbry.io/faq/naming
+4. Once your name is selected, there is a `Deposit` section that is below.  It requires a minimum bid of .1 LBC.   See more on deposits here: naming.md.  If you would ever like to abandon your claim there is a guide below in `Abandoning a claim`.
+5. Click `Create Channel` once you have selected your bid amount.  Your deposit is a bid and if this channel isn't claimed by another bid, the channel is yours to use!
+You can revoke this claim if you no longer want to use it with the steps below.
+
+### Publish Content to a Channel
+
+1. Under the `Content` section click `Choose File`.
+2. Select what you would like to upload to LBRY.  LBRY accepts any HTML5 format for video, the full list can be found here: https://developer.mozilla.org/en-US/docs/Web/HTML/Supported_media_formats
+3. Select a `Title`, `Thumbnail URl`, and `Description` for your upload.
+4.  Below those there is a `Language` and `Maturity` which will default to `English` and `All Ages`.  If a change is needed to these defaults click the dropdowns and change them.
+Under the `Access` section select what type of license the content should be under, select which type of license is appropriate for your upload. </br>
```

Comment:
select, select again - wording
Can you find anything online that might explain these copyright options?